examples/enterprise_edr/threat_intelligence/README.md

```diff
@@ -1,6 +1,8 @@
 # ThreatIntel Module
 Python3 module that can be used in the development of Threat Intelligence Connectors for the Carbon Black Cloud.
 
+> **_NOTE:_** This connector is being deprecated, it is recommended to use the `cbc-stix-taxii-connector` instead!
```

Comment:
This has to be updated with the link to the repository instead that's why its in [WIP]

Reply:
I'll approve once the link is available and the language is changed from present to past tense